pyproject.toml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
The copyright seems to be repeated...

Reply:
Thanks for catching this, likely during a copy and paste. Fixed in all PRs.